src/pseudolik.jl

```diff
@@ -1291,19 +1310,21 @@ end
 function calculateExpCFAll!(data::DataCF, x::Vector{Float64},net::HybridNetwork)
     !all((q->(q.qnet.numtaxa != 0)), data.quartet) ? error("qnet in quartets on data are not correctly updated with extractQuartet") : nothing
     #println("calculateExpCFAll in x: $(x) with ht(net) $(ht(net))")
-    for q in data.quartet
-        update!(q.qnet,x,net)
-        if(q.qnet.changed)
-            #println("enters to recalculate expCF for some quartet")
-            qnet = deepcopy(q.qnet);
-            calculateExpCFAll!(qnet);
-            q.qnet.expCF[:] = qnet.expCF
+    Threads.@threads for q in data.quartet
+        if (q.sampled)
+            update!(q.qnet,x,net)
+            if(q.qnet.changed)
+                #println("enters to recalculate expCF for some quartet")
+                qnet = deepcopy(q.qnet);
+                calculateExpCFAll!(qnet);
+                q.qnet.expCF[:] = qnet.expCF
+            end
         end
     end
 end
 
 """
-    topologyQpseudolik!(net::HybridNetwork, d::DataCF)
+    `topologyQpseudolik!(net::HybridNetwork, d::DataCF)`
```

Comment:
Why these backticks? There should be 4 spaces in front of the function signature, no backticks.